compose-builder/docker-compose-base.yml

```diff
@@ -35,8 +35,6 @@ services:
     image: hashicorp/consul:${CONSUL_VERSION}
     command: "agent -ui -bootstrap -server -client 0.0.0.0"
     user: "root:root" # Note that Consul is run under the 'consul' user, but entry point scripts need to first run as root
-    ports:
```

Comment:
With your new discovery these can stay and add `ports: !reset null` to the zero-trust yaml?

Reply:
yep. pushed that change up